calico-enterprise/visibility/visualize-traffic.mdx

```diff
@@ -2,7 +2,89 @@
 description: Learn the power of network sets.
 ---
 
-# Visualize traffic to and from a cluster
+# Network Visualization
+> From the left navbar, select **Service Graph**, **Default**
+
+Service Graph offers tools to visualize and investigate network communications in the cluster. To better understand how
+Service Graph works, here is the list of core features and components.
+
+## Service Graph Features and Components
+
+**Namespaces**
+
+Namespaces are the default view in Service Graph.
+
+When you expand the top right panel `<<`, you see a detailed view of the service-to-service communications for the namespace.
```

Comment:
This requires to first click on a node or edge, is that right?

Reply:
@rene-dekker this is actually not accurate, we dont see details of service-to-service communications in the top tight panel by clicking on a namespace, we rather see details about inbound / outbound to other namespaces and list of services that belong to the namespace. So how about replacing this with:

```
When you click on a namespace to expand the top right panel `<<`, you see a detailed information about the to / from namespaces and services that belong to the selected namespace. For detailed information about service-to-service communications in the namespace, we can double click on the namespace node.
```